dbtestutil_test.go

```diff
@@ -80,11 +85,12 @@ type rSliceElementForTesting struct {
 	IsUndefined bool
 }
 
-func (s *DbZuite) DbState() ([]rWorksheet, []rValueForTesting, []rSliceElementForTesting) {
+func (s *DbZuite) DbState() ([]rWorksheet, []rValueForTesting, []rParent, []rSliceElementForTesting) {
```

Comment:
This return tuple is getting pretty big. What's the rule of thumb for Golang on when to convert to a struct?